torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -49,13 +49,22 @@ Operation BroadOp(const Node* node) {
       push(stack, a, b);
     } else {
       auto out_size = at::infer_size(a_size, b_size);
+      size_t out_numel = out_size[0];
```

Comment:
I don't if we need to worry about 0-dim case here?

Reply:
no, we dont fuse 0-dim tensors, we handle tensors with a dimension value of 0